jsonparse.ps1

```diff
@@ -0,0 +1,83 @@
+$location=$args[0]
+$repo=$args[1]
+$name=$args[2]
+
+$location
+$repo
+$name
+$repo.Title
+$JSON = (Get-Content $location) -join "`n" | ConvertFrom-Json
```

Comment:
Naming:

1. Since all your other variable names are camelCase (which agrees with PowerShell [coding conventions](https://github.com/PoshCode/PowerShellPracticeAndStyle/issues/36) that I've seen in various places, let's use that convention for this variable as well ("`$json`") and also for "`$uri`".

2. But this particular blob of JSON represents a SARIF log, so let's name it either `$sarifLog` or `$log`. (I'd prefer `$sarifLog` because that's the name of the class in the SARIF object model: [SarifLog.cs](https://github.com/Microsoft/sarif-sdk/blob/master/src/Sarif/Autogenerated/SarifLog.cs).